aws/scalardl/example.tfvars

```diff
@@ -13,21 +13,17 @@ scalardl = {
   # green_image_tag              = "2.1.0"
   # green_image_name             = "ghcr.io/scalar-labs/scalar-ledger"
   # green_discoverable_by_envoy  = "false"
+  # container_env_file           = "scalardl_container.env"
```

Comment:
The file names don't match, rename and use it? `example.scalardl_container.env`

Reply:
It's assumed that the `scalardl_container.env` will be copied from `example.scalardl_container.env`.

Reply:
Can we use the `scalardl_container.env` file directly? is there any problem?

Reply:
It's possible, but since the user must change the contents of the `env` file, if a file `scalardl_container.env` is in the repo, the local git state will be dirty as the user changes the content.
I'd like to add the `.gitignore` lines like the following (just like `example.tfvars` is tracked by Git while `*.tfvars` is ignored). What do you think?

```
*.env
!example.*.env
```